release/release.js

```diff
@@ -325,7 +324,7 @@ function checkoutMasterAndMergeDev() {
             `git checkout ${config.devBranch}`,
             `git checkout ${config.masterBranch}`,
             `git submodule sync`,
-            `git submodule update`,
+            `git submodule update --init`,
```

Comment:
You probably want `--recusive` for good measure?

Reply:
Not really we only update the pointer of a submodule not of the submodule's submodule (that would be done in that submodule repo). So doing --recursive would just slow down the process.